src/sentry/eventstream/kafka/backend.py

```diff
@@ -137,18 +144,39 @@ def _send(
     def requires_post_process_forwarder(self):
         return True
 
+    def reset_batch(self) -> None:
+        self._pending_futures.clear()
+        self._batch_create_time_ms = int(time.time() * 1000)
+
+    @staticmethod
```

Comment:
Why is this a staticmethod?

Reply:
This method does not depend on the object or class. Hence made it a staticmethod

Reply:
That makes sense.